corehq/apps/reports/static/reports/js/case_list.js

```diff
@@ -1,9 +1,26 @@
+hqImport("hqwebapp/js/initial_page_data").get('user_types')
 hqDefine("reports/js/case_list", ['jquery', 'analytix/js/kissmetrix'], function ($, kissAnalytics) {
     $(function () {
         if (document.location.href.match(/reports\/case_list/)) {
             $(document).on('click', 'td.case-name-link', function () {
                 kissAnalytics.track.event("Clicked Case Name in Case List Report");
             });
+            var user_types = hqImport("hqwebapp/js/initial_page_data").get('user_types');
+            $(document).on('click', 'button#apply-filters', function () {
+                kissAnalytics.track.event("Clicked Apply",
+                    {"filters": _.map(_.find($('#paramSelectorForm').serializeArray(),
```

Comment:
Are you sending all filters or just the case owners? The trello card reads to me like it wants all filters, but this looks like it's just the owner being sent.

If you're just picking out the value of the owners filter, you can select it directly with something like `$("#paramSelectorForm input[name='case_list_filter']")` rather than gathering all of the form's inputs and searching through them. And the property name should be more specific than "filters" - "case_list_filter" is probably right, since then it'll match the backend code.

Reply:
One more thing: looks like you're sending an array of owner names. Maybe kissmetrics will handle that properly, but I'd `join` the into array into a comma-separated string to be safe.

Reply:
On the first point, I messaged Dev asking him what he wanted, 
And for the second point: https://github.com/dimagi/commcare-hq/pull/22079/commits/feddd1082da7e4b2c58c7e47a3afe3d877c6b83e (that is a million times better, thanks)

Reply:
Oh and I missed the `join` thing the first time I looked at this, added that in here: https://github.com/dimagi/commcare-hq/pull/22079/commits/8912edf058e8e469323aeb113a96058bf512c553